locallib.php

```diff
@@ -363,6 +382,11 @@ function local_bulkenrol_users($localbulkenrolkey) {
     }
 }
 
+/**
+ * According to the parameter, either a table with hints is displayed or a table with users to be written is displayed.
+ * @param unknown $localbulkenroldata
+ * @param unknown $key
```

Comment:
There is no param type "unknown" - see https://docs.phpdoc.org/guides/types.html. Can you clarify the type here?